lib/_stream_readable.js

```diff
@@ -12,6 +12,25 @@ var StringDecoder;
 
 util.inherits(Readable, Stream);
 
+// This does not use the internal/errors so that it can remain
+// portable with the readable-streams module.
```

Comment:
probably also point to these from the main functions in case someone tries to edit them without knowing these exist?